__tests__/app.test.js

```diff
@@ -22,4 +22,21 @@ describe('GET /api/topics', () => {
             });            
         })
     });
+});
+
+describe('GET /api', () => {
+    it('should respond with all other endpoints and their description', () => {
```

Comment:
Please see Rose's post in the main channel on how to address this endpoint in a more straightforward way.